src/config-schemas/vitals-config-schema.ts

```diff
@@ -0,0 +1,49 @@
+import { validators } from "@openmrs/esm-module-config";
+
+export const vitalsConfigSchema = {
+  triageFormConfig: {
```

Comment:
What's the triage form?

Reply:
It's the encounter type and formuuid that is used to with the vitals form. Right now it's hardcoded and hence the form can't save in other distributions.